_config.yml

```diff
@@ -135,8 +135,8 @@ t:
         content: "Le centre national d'intelligence artificielle chilien avec qui on travaille pour notre premier déploiement à l’étranger"
       partner4:
         title: "ADF Subirats"
-        url: "https://sfadf.org/adf/subirats//"
-        logo: "https://sfadf.org/wp-content/themes/sfadf/img/logo-header.jpg"
+        url: "https://subirats.cat/adfsubirats/"
+        logo: "../img/logo-adf-subirats.png"
```

Comment:
This is not the same website nor the right logo. Why did you change it if I may?

Reply:
I changed it because what is now on the web is not correct. The logo and the link that have been uploaded now are those that represent our organization and take advantage of this PR to change them.

Reply:
Oh, perhaps adding a description would help then, because I had no context about this PR :sweat_smile: